packages/xo-web/src/xo-app/host/tab-advanced.js

```diff
@@ -112,6 +113,7 @@ export default class extends Component {
       return uniqPacks
     }
   )
+  _setHostIscsiIqn = iscsiIqn => editHost(this.props.host, { iscsiIqn })
```

Comment:
According to [the documentation](https://docs.citrix.com/en-us/xenserver/7-0/downloads/administrators-guide.pdf) the IQN must be unique. Please take care of this warnings
![image](https://user-images.githubusercontent.com/21563339/58262492-ad63ed00-7d7a-11e9-8dd9-7441a96ab438.png)